c3pzero_ignition/README.md

```diff
@@ -0,0 +1,11 @@
+# c3pzero_ignition
+
+Ignition Gazebo simulation related launch, world and environment hooks for c3pzero robot. This package contains two launch files:
+
+- ignition.launch.py: Launches a given world file in Ignition Gazebo Simulator, defaults to empty_world.sdf. To launch:
+
+  ros2 launch c3pzero_ignition ignition.launch.py world:=empty
+
```

Comment:
```suggestion
```
  ros2 launch c3pzero_ignition ignition.launch.py world:=empty
```
```

Reply:
These aren't rendering right but for code examples let's wrap them in a code block